BEST_PRACTICES.md

```diff
@@ -266,6 +267,42 @@ For example, lets say you have a method that returns a filename for a photo, in
 
 In cases where the crash is obvious such as when dequeuing cells and casting them to your specific cell class, force unwrapping is preferred, because if you can't dequeue the cell, or can't a cell for the specific class, there's no other way to recover than to show an empty screen, which is not preferable.
 
+## View properties and lazy-loading
```

Comment:
Ops, was supposed to be `class properties` although that can be misunderstood. Just `properties`?